test/java/synth-app/__snapshots__/test.ts.snap

```diff
@@ -24,6 +25,18 @@ exports[`java full integration synth generates JSON 1`] = `
         \\"version\\": \\"~> 3.1.0\\",
         \\"source\\": \\"hashicorp/random\\"
       }
+<<<<<<< HEAD
```

Comment:
Here are a bunch of merge markers, maybe you can clean the commit up

Reply:
done!